### ISSUE TITLE
Keyboard focus going outside of the ‘Pages’ opened popup without closing it.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#104]
User Experience:

If focus order is not sequential and meaningful, then keyboard user will find hard to understand the actual flow of tab and the actual feature of the page which degrades the user.

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.​
3. Wait portal to load in design mode​
4. Navigate to the Left nav and select Pages button.​
5. Navigate to the various controls of the "Pages" window using keyboard
6. Verify the issue.
 

Actual Result:

Keyboard focus going outside of the ‘Pages’ opened popup without closing it and start from top of the page, while navigating with keyboard, which is not logical.

 

Expected result:

Keyboard focus should be logical and should not move to outside of the ‘Pages’ opened popup  without closing it, keyboard focus should remain in side popup and after ‘Add pages’ button keyboard focus should moves to help icon button.

 

Note:

Similar issue exist with Chrome.

Similar issue exist throughout the application.

 

MAS Reference:

MAS 2.4.3 - Focus Order (05)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
FIxed.
